components/input.css

```diff
@@ -0,0 +1,33 @@
+
+ .input{
+     width: 80%;
+     padding: 10px;
+     height: 1rem;
```

Comment:
Redundant code. The default height of input is 1rem only.

Reply:
+1